test/conftest.py

```diff
@@ -0,0 +1,133 @@
+'''
```

Comment:
It might be cleaner to still have this in `shared/utils.py` and just import the right thing from `utils` in `conftest.py`?

I don't know what else we might want to add to `conftest.py`, but I think it's nice if the file itself is kept short and readable and just delegates its content to other files?

Reply:
`conftest.py` is place where pytest looks for fixtures.

Reply:
I know that.

But you can still implement these fixtures in another file and just import/expose them in `conftest.py`. See https://github.com/YannickJadoul/Parselmouth/blob/7c4ea044c3894763397b5dd103b4550ade278f88/tests/conftest.py#L34 for example.

Reply:
You're right, it would be cleaner.

Reply:
Done